R-packages/evalcast/tests/testthat/test-manage_cards.R

```diff
@@ -0,0 +1,18 @@
+test_that("scale_by_forecaster works", {
+    score_card <- tibble(
+        forecaster = c("f1", "f2", "f1", "f2"),
+        ahead = c(1, 1, 2, 2),
+        coverage_80 = c(1, 1, 0, 0),
+        ae = c(1, 2, 4, 16),
+        wis = c(3, 5, 7, 9)
+    )
+    scaled_card <- scale_by_forecaster(score_card, c("ae", "wis"), "f2")
```

Comment:
Maybe it's already tested (I can't quite tell), but if not, can you (1) test the checks I suggested above and (2) test that "Elements of `err_cols` that are not in `vars` will be dropped"?

Reply:
Done.